src/java/TransactionPermutation.java

```diff
@@ -9,14 +9,23 @@
 /**
  * Created by snakecharmer1024 on 7/20/16.
  */
-public class TransactionPermutation
-{
-    public Sha256Hash txHash;
-    public Peer entryPeer;
-    public List<Integer> permutation;
+
+// This class is created to track the ordinality of PeerGroup 
+// transmissions with regard to a specific transaction.
+
+public class TransactionPermutation {
+
+    public Sha256Hash txHash; //SHA256 encoded transaction information
+    public Peer entryPeer; // is entryPeer self? (ie entry node of network)
```

Comment:
This is changed in the next PR, this part of the code doesn't really need to know about the entryPeer, it just needs to have the data label. EntryPeer is not self, it's the first node we send that transaction to, which is what the clustering will try to infer.